<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>733</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>20</horstretch>
     <verstretch>20</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="in">
       <attribute name="title">
        <string>Ввод</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="Widget3d" name="mainWindow" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>275</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="inputButton">
             <property name="text">
              <string>Ввод области</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="textInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkGauss">
             <property name="text">
              <string>Точки схемы Гаусса</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkOnlyLines">
             <property name="text">
              <string>Только фигура</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonReDo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Изменить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="out">
       <attribute name="title">
        <string>Решение</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGroupBox" name="rcvrsGroupBox">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="title">
               <string>Прямая задача</string>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0" colspan="2">
                <widget class="QPushButton" name="execDirectButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Выполнить</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0" colspan="2">
                <widget class="QLabel" name="textNum">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Введите количество разбиений</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::AutoText</enum>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="labelX">
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item row="8" column="1">
                <widget class="QSpinBox" name="numYBox">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QSpinBox" name="numXBox">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                </widget>
               </item>
               <item row="8" column="0">
                <widget class="QLabel" name="labelY">
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="grvtGroupBox">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="title">
               <string>Обратная задача</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QPushButton" name="execInverseButton">
                 <property name="text">
                  <string>Выполнить</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkIsReg">
                 <property name="text">
                  <string>Использовать регуляризацию</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="alphaLabel">
                   <property name="text">
                    <string>Введите альфа</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="alphaBox">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="decimals">
                    <number>10</number>
                   </property>
                   <property name="singleStep">
                    <double>0.000001000000000</double>
                   </property>
                   <property name="value">
                    <double>0.000001000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="gammaLabel">
                   <property name="text">
                    <string>Введите гамма</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="gammaBox">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="decimals">
                    <number>10</number>
                   </property>
                   <property name="singleStep">
                    <double>0.000001000000000</double>
                   </property>
                   <property name="value">
                    <double>0.000001000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="labelGammaD">
                   <property name="text">
                    <string>Шаг по гамме</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="gammaDBox">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="decimals">
                    <number>10</number>
                   </property>
                   <property name="singleStep">
                    <double>0.000001000000000</double>
                   </property>
                   <property name="value">
                    <double>0.000001000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="labelPercentage">
                 <property name="text">
                  <string>Возможное изменение функционала</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="percBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="decimals">
                  <number>1</number>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelIter">
                 <property name="text">
                  <string>Количество иттераций:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labIter">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="wdgtGroupBox">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="title">
               <string>Параметры отображения</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QCheckBox" name="checkBoxX">
                   <property name="text">
                    <string>x</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBoxY">
                   <property name="text">
                    <string>y</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBoxZ">
                   <property name="text">
                    <string>z</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QSlider" name="verticalSliderX">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="verticalSliderY">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="verticalSliderZ">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QRadioButton" name="radAxis">
                 <property name="text">
                  <string>Отображать оси</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="Widget3dSecond" name="mainWindow2" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Widget3d</class>
   <extends>QWidget</extends>
   <header>widget3d.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Widget3dSecond</class>
   <extends>QWidget</extends>
   <header>widget3dsecond.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
